### ISSUE TITLE
Error.captureStackTrace is not a function

### ISSUE THREAD

**Issue by bramn** [janniks/vue-notion#40]
I am trying to run the example locally on my laptop, but I keep bumping into this error message:
```
Error.captureStackTrace is not a function
An error occurred while rendering the page. Check developer tools console for details.
```

Is the current version of `vue-notion` broken? Possibly due to changes in the Notion.so API?

What I did:
- Clone this project
- Change directory to `example`
- `npm i`
- `npm run dev`
- Go to http://localhost:3000 in my browser
- Observe the above error message...

I am not that familiar with Nuxt/Vue to know how to debug this only error message ☺️

**Comment by janniks** [comment]
Thanks for the report, I will check as well 👍🏻

**Comment by janniks** [comment]
Hmm, if works in a clean clone for me.

Which node version are you running? `node -v`

**Comment by janniks** [comment]
I think I found the bug. I just released `1.1.2` with a different build script — could you update and check if the error persists?

**Comment by bramn** [comment]
I checked and it seems to be working fine now! Thanks 😊
